chart/values.yaml

```diff
@@ -65,6 +65,8 @@ ingress:
 letsEncrypt:
   # email: none@example.com
   environment: production
+  # options: nginx, traefik
+  ingressClass: nginx
```

Comment:
May be it's better add class variable nested to ingress , `ingress.class` instead of `letsEncrypt.ingressClass` ??